### ISSUE TITLE
Show phone number of contact in follow-up visits/ new visit

### ISSUE THREAD

**Issue by max-hzi** [hzi-braunschweig/SORMAS-Project#2318]
### Situation Description
The phone number of the contact is documented in 'contact person', but the phone call is tracked in a different tab/ pop-up window. 

### Feature Description
The phone number should show in the pop-up window 'create new visit', maybe at the top right (see screenshot), so the info is at the hand of the person who makes the call. The phone number should not be editable.
![1224](https://user-images.githubusercontent.com/63781233/84644951-31e25a00-af00-11ea-861a-f139ec84c988.PNG)

**Comment by bernardsilenou** [comment]
Comment from #2716 
- Use description or caption to indicate that this is the contact persons phone number
- Add phone number to followup visit forms for contact and also for cases. We will do the same for event participants in another issue and only after implementation of followup for event participants

**Comment by tabotkevin** [comment]
@bernardsilenou following you statement here "Add phone number to followup visit forms for contact and also for cases.", I can't seem to find where followup visit forms for cases is located when I logged into the system. A little visual help will be appreciated. I didn’t have any problems finding followup visit forms for contact.

**Comment by bernardsilenou** [comment]
@tabotkevin Sorry for that, It will be included in the 1,47 that will be release tomorrow afternoon
